estate/models/estate_property.py

```diff
@@ -0,0 +1,95 @@
+from odoo import models, fields, api
+from dateutil.relativedelta import relativedelta
+from odoo.exceptions import UserError, ValidationError
+from odoo.tools.float_utils import float_compare
```

Comment:
```suggestion
from dateutil.relativedelta import relativedelta

from odoo.exceptions import UserError, ValidationError
from odoo import models, fields, api
from odoo.tools.float_utils import float_compare
```

First external libraries and then internal import.